### PR TITLE
Added the steps stack into the Fonda Context

### PR DESCRIPTION
**Pull request by ElChache**
Added the steps stack into the Fonda Context so the loggers can have access to the executed steps.

### CODE REVIEW

File: src/fonda/core.cljs [hunk 1]
```diff
@@ -71,6 +71,7 @@
            :on-anomaly    on-anomaly
            :on-exception  on-exception
            :queue         (st/steps->queue steps)
+           :stack         []
```
Comment (mattbishop) [inline src/fonda/core.cljs]: Why should stack and queue need to be visible? Seems like a dangerous thing.

Comment (ElChache) [reply src/fonda/core.cljs]: stack is only visible to the loggers, whose result is ignored

File: README.md [hunk 2]
```diff
@@ -38,8 +38,8 @@ The parameter order makes it easy to partially apply `execute` for leaner call s
 | Key | Optional? | Notes |
 |---|---|---|
 | `:anomaly?` | Yes | A function that gets a map and determines if it is an anomaly |
-| `:log-exception` | Yes | A function that gets called with the [fonda context](#fonda-context) when there is an exception |
-| `:log-anomaly` | Yes | A function that gets called with the [fonda context](#fonda-context) when a step returns an anomaly |
+| `:log-exception` | Yes | A function that gets called with the [fonda context](#logging) when there is an exception |
```
Comment (arichiardi) [inline README.md]: called with the `log-map`?

File: test/fonda/loggers_test.cljs [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                     :processor (fn [_] (throw thrown-exception))}
+          log-exception (fn [{:keys [stack exception]}]
+                          (is (= processor (select-keys (last stack) [:path :name :processor])))
+                          (is (= exception thrown-exception))
+                          (done))]
+      (fonda/execute {:log-exception log-exception}
+                     [processor] {}
+                     success-cb-throw
+                     anomaly-cb-throw
+                     (fn [_])))))
```
Comment (arichiardi) [inline test/fonda/loggers_test.cljs]: is this doing anything significant?

File: test/fonda/loggers_test.cljs [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+                     :name      "processor name"
+                     :processor (fn [_] returned-anomaly)}
+          log-anomaly (fn [{:keys [stack anomaly]}]
+                        (is (= processor (select-keys (last stack) [:path :name :processor])))
+                        (is (= anomaly returned-anomaly))
+                        (done))]
+      (fonda/execute {:log-anomaly log-anomaly}
+                     [processor] {}
+                     success-cb-throw
+                     (fn [_])
```
Comment (arichiardi) [inline test/fonda/loggers_test.cljs]: is this doing anything significant?

File: test/fonda/loggers_test.cljs [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+    (let [processor {:path      [:processor-path]
+                     :name      "processor name"
+                     :processor (fn [_] :a)}
+          log-success (fn [{:keys [ctx stack]}]
+                        (is (= ctx {:processor-path :a}))
+                        (is (= processor (select-keys (last stack) [:path :name :processor])))
+                        (done))]
+      (fonda/execute {:log-success log-success}
+                     [processor] {}
+                     (fn [_])
```
Comment (arichiardi) [inline test/fonda/loggers_test.cljs]: is this doing anything significant?

**Pull request merged**